scripts/reportportal/rp_manual_reporter/rp_manual_reporter.py

```
@@ -0,0 +1,760 @@
+
+    Raises:
+        click.ClickException: If the file format is invalid or a status
+            value is unrecognised.
+    """
+    try:
+        raw = yaml.safe_load(batch_path.read_text(encoding="utf-8"))
+    except (OSError, yaml.YAMLError) as exc:
+        raise click.ClickException(f"Failed to read batch file {batch_path}: {exc}") from exc
+
+    if not isinstance(raw, dict) or "results" not in raw:
+        raise click.ClickException(f"Batch file {batch_path} must contain a top-level 'results' key")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)